packages/@adobe/spectrum-css-temp/components/dropdown/index.css

```diff
@@ -170,3 +170,7 @@ governing permissions and limitations under the License.
     flex-direction: row;
   }
 }
+
+.spectrum-Dropdown-popover {
+  overflow: hidden;
+}
```

Comment:
Is the ComboBox/SearchAutocomplete popover supposed to have this same `overflow: hidden` as well? Still seeing the following corner defects:
<img width="364" alt="image" src="https://github.com/adobe/react-spectrum/assets/9661127/ca0ffa1e-c736-4dc0-86dc-88caf3009876">

Reply:
It should probably have the same `overflow: hidden` to fix the corner defects.

Reply:
See https://github.com/adobe/react-spectrum/commit/e035bc55d6349ade4e130bea2de8f077bd7ab0b7

Reply:
I think we can avoid the need for this. In e4b05779ef209fa500e26cf899d82057b4cd3b38 I made it so popovers are `overflow: hidden` by default, except if they have an arrow. In that case, the dialog inside is `overflow: hidden` as well. That also fixes the issue with overflow that you had added the extra padding for at 400px breakpoint. The dialog itself should never scroll, because either the content or the grid inside does. The `max-height: inherit` that was there before was inheriting the height including the borders, thus off by 2px, so getting rid of that and adding `overflow: hidden` fixes it.